### PR TITLE
Update engagement form styles

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #1068 :*
https://github.com/bcgov/met-public/issues/1068

-Add softer borders to rich content editor

-Add softer borders to engagement text fields

-fix tab spacing

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-web/src/components/engagement/form/RichTextEditor.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ const RichTextEditor = ({
 
     return (
         <FormControl fullWidth>
-            <MetPaper style={{ borderColor: `${error ? '#d32f2f' : '#606060'}` }}>
+            <MetPaper style={{ borderColor: `${error ? '#d32f2f' : ''}` }}>
```
Comment (jadmsaadaot) [inline met-web/src/components/engagement/form/RichTextEditor.tsx:63]: why? Don't we want this border to be red when there's error?

File: met-web/src/components/engagement/form/EngagementWidgets/Phases/PhasesOptionCard.tsx [hunk 2]
```diff
@@ -16,6 +16,8 @@ export const optionCardStyle = {
     cursor: 'pointer',
     '&:hover': { backgroundColor: 'rgb(242, 242, 242)' },
     minHeight: '110px',
+    border: '1px solid #cdcdcd',
+    boxShadow: 'rgb(0 0 0 / 6 %) 0px 2px 2px - 1px, rgb(0 0 0 / 6 %) 0px 1px 1px 0px, rgb(0 0 0 / 6 %) 0px 1px 3px 0px',
```
Comment (jadmsaadaot) [inline met-web/src/components/engagement/form/EngagementWidgets/Phases/PhasesOptionCard.tsx:20]: If we do that in StyledPaper, do we need to it here?

File: met-web/src/components/engagement/form/RichTextEditor.tsx (hunk 1 above)
Comment (djnunez-aot) [reply met-web/src/components/engagement/form/RichTextEditor.tsx:63]: the border is still red when there is an error it is just going to inherit the other border style when there is no error

File: met-web/src/components/engagement/form/EngagementWidgets/Phases/PhasesOptionCard.tsx (hunk 2 above)
Comment (djnunez-aot) [reply met-web/src/components/engagement/form/EngagementWidgets/Phases/PhasesOptionCard.tsx]: good catch no we do not!

File: met-web/src/components/engagement/form/RichTextEditor.tsx (hunk 1 above)
Comment (jadmsaadaot) [reply met-web/src/components/engagement/form/RichTextEditor.tsx:63]: ohhh right! Then let's something like sx={[ error && { borderColor:...}]}. This way borderColor is not set to a bad value like '' when there's no error
image

**Review by jadmsaadaot — APPROVED** [review]

**Review by VineetBala-AOT — APPROVED** [review]

**Pull request merged**